Tools/CISupport/ews-build/steps.py

```diff
@@ -80,6 +80,12 @@ class GitHub(object):
     def repository_urls(cls):
         return [GITHUB_URL + project for project in GITHUB_PROJECTS]
 
+    @classmethod
+    def user_for_queue(cls, queue):
+        if queue.lower() in ['unsafe-merge-queue', 'merge-queue']:
+            return 'merge-queue'
```

Comment:
Can return generic name here, e.g.: `GITHUB_PRIORITY_QUEUES` / `GITHUB_PRIORITY` / `GITHUB_GROUP1`

Reply:
We could, but then we have to change the key in our passwords file, I think we want this to cause us to use the default name if we return `None`